pkg/statistics/handle/autoanalyze/priorityqueue/calculator.go

```diff
@@ -56,6 +56,13 @@ func (pc *PriorityCalculator) CalculateWeight(job AnalysisJob) float64 {
 	// when we apply the log10 function, the resulting value is more meaningful and reasonable.
 	indicators := job.GetIndicators()
 	changeRatio := 100 * indicators.ChangePercentage
+	if j, ok := job.(*DynamicPartitionedTableAnalysisJob); ok {
+		tblSize := indicators.TableSize * (1 + float64(len(j.SpecialGlobalIndexes)))
```

Comment:
I am not sure why we need to use `indicators.TableSize(rows * cols)` * (1 + SpecialGlobalIndexes) here. I guess it would be rows * (normal cols count + SpecialGlobalIndexes count)?